address/address.go

```diff
@@ -185,7 +185,8 @@ func ToBech32(bc *Bech32) (string, error) {
 }
 
 // FromBase58Confidential decodes a confidenail address that was base58 encoded
-//  and verifies the checksum.
+//
+//	and verifies the checksum.
```

Comment:
can you revert this? I guess your purpose was to remove the extra leading space.